node-problem-detector-0.8.yaml

```diff
@@ -1,13 +1,15 @@
 package:
   name: node-problem-detector-0.8
   version: 0.8.20
-  epoch: 8
+  epoch: 9
   description: node-problem-detector aims to make various node problems visible to the upstream layers in the cluster management stack.
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
       - busybox
+      - health-checker-0.8=${{package.full-version}}
```

Comment:
please use `major-minor-version` variable for 0.8 part otherwise on updates to 0.9 the package will get auto-merged by the bot with wrong runtime deps.